resources/migrations/008-create-items.edn

```diff
@@ -0,0 +1,12 @@
+;; resources/migrations/008-create-items.edn
+{:up   ["CREATE TABLE items (
+          id SERIAL PRIMARY KEY,
+          category_id INTEGER REFERENCES categories ON DELETE CASCADE NOT NULL);
+         CREATE TABLE field_values (
+           id          SERIAL PRIMARY KEY,
+           item_id     INTEGER REFERENCES items ON DELETE CASCADE NOT NULL,
+           field_id    INTEGER REFERENCES fields ON DELETE CASCADE NOT NULL,
+           category_id INTEGER REFERENCES categories ON DELETE CASCADE NOT NULL,
```

Comment:
Isn't this association redundant? 
field_values have a foreign key on items, which in turn have a foreign key on categories.

Reply:
`category_id` is added to `field_values` to optimise for lookup by category (which is very frequent, it would otherwise require a join). Not sure if the foreign key constraint needs to be present in `field_values` as well. Thoughts pls?